cmake/FDBComponents.cmake

```diff
@@ -46,7 +46,8 @@ endif()
 set(WITH_JAVA OFF)
 find_package(JNI 1.8)
 find_package(Java 1.8 COMPONENTS Development)
-if(JNI_FOUND AND Java_FOUND AND Java_Development_FOUND)
+# leave FreeBSD JVM compat for later
```

Comment:
Could you file an issue for it, and reference it here?

Reply:
https://github.com/apple/foundationdb/issues/3054 done & added it in the forum so I don't lose it ;-). If you can assign that issue to me that would be even better.